conda-forge.yml

```
@@ -4,7 +4,7 @@ os_version:
   linux_aarch64: cos7
   linux_ppc64le: cos7
 provider:
-  linux_aarch64: default
+  linux_aarch64: azure
   linux_ppc64le: azure
 build_platform:
   linux_ppc64le: linux_64
```

Guessing we are looking to use the cross-compilation build mode for these. If so, would suggest the following

```suggestion
provider:
  linux_aarch64: default
  linux_ppc64le: default
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

---

Yeah let's check what happened with the  compilation test first (since it failed in both native and emulated CIs), before switching to cross-compilation mode (which would not run the compilation tests, see the ppc CIs) to shorten the CI time.

---

Think we could configure a compilation run in the test phase to work with cross-compilation. We just wouldn't be able to run the code (though that is likely true even for a native build if it needs a GPU)

---

@jakirkham I’m not following how this config would trigger cross-compile.

Is there some documentation on the subject you can share?